src/components/sidenav/demoClickOutsideToClose/script.js

```diff
@@ -0,0 +1,11 @@
+angular
+  .module('sidenavDemo3', ['ngMaterial'])
```

Comment:
I would like to get rid of this `sidenavDemoX` pattern and name the module after the demo, i.e. `sidenavDisableCloseEvents`.

Reply:
What do you think about a pattern like this 'customSidenavDemo', 'basicSidenavDemo', 'disableCloseEventsSidenavDemo' ?

Reply:
Sounds good!